web/src/lib/roadmap/buildUserRoadmap.test.ts

```diff
@@ -184,6 +184,23 @@ describe("buildUserRoadmap", () => {
         await buildUserRoadmap(generateStartingProfile({ industryId: "food-truck" }));
         expect(getLastCalledWith(mockRoadmapBuilder)[0].addOns).not.toContain("permanent-location-business");
       });
+
+      it("does not add env-permitting add-on if home-based business is true", async () => {
```

Comment:
dust - i would include the generic industry in the title since the functionality is dependent on it.